website/docs/reference/deploy/configuring-unleash.md

```diff
@@ -193,7 +193,7 @@ Some facets of the [segments feature](../segments.mdx) can be customized via env
 
 `UNLEASH_STRATEGY_SEGMENTS_LIMIT` controls the maximum number of segments that can be applied to a single strategy. The default is 5.
 
-`UNLEASH_SEGMENT_VALUES_LIMIT` controls the maximum number of values that you can assign across a segment's constraints. The default is 100.
+`UNLEASH_SEGMENT_VALUES_LIMIT` controls the maximum number of values that you can assign across a segment's constraints. The default is 1000 (Since v5.1.0)
```

Comment:
I suspect that at least for a while, people who aren't on v5.1.0 will want to know what it was before, so maybe add a note like this?

```suggestion
`UNLEASH_SEGMENT_VALUES_LIMIT` controls the maximum number of values that you can assign across a segment's constraints. The default is 1000 (Since v5.1.0; it was 100 before v5.1.0)
```